install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -384,16 +384,16 @@ spec:
             value: {{ .Values.cgroup.hostRoot }}
```

Comment:
please use `Reported-By: André Martins <andre@cilium.io>` and add `Fixes: fa8bea45562f ("cilium-daemonset: Fix ineffective socket-lb caused by incorrect cgroup2 fs mount")` in the commit message